validation/elemental/schemas/elemental_schemas.yaml

```diff
@@ -75,9 +96,19 @@
       steps:
         - position: 1
           action: "Create Rancher"
-          data: ""
-          expectedresult: "Rancher instance is up and accessible"
+          expectedresult: "Rancher Server is up and accessible"
```

Comment:
Rename this to Rancher Manager. You are just saying that the Rancher UI should be available for use, right?